src/dyld/darling

```diff
@@ -66,8 +66,19 @@ setup_prefix() {
     rm -f "$1/dev"
     ln -sf "system-root/dev" "$1/dev"
 
+    rm -rf "$1/sandboxed-home"
+
     rm -f "$1/home"
```

Comment:
True. However if any mkdir fails, function returns prematurely.
I'll add checks for $1 and replace $1 usage with local variable.